src/Microsoft.Extensions.Caching.SqlConfig.Tools/Program.cs

```diff
@@ -124,14 +124,33 @@ private void CreateTableAndIndexes()
                         transaction.Commit();
 
                         _logger.LogInformation("Table and index were created successfully.");
+
+                        return 0;
                     }
                     catch (Exception ex)
                     {
-                        _logger.LogError("An error occurred while trying to create the table and index.", ex);
+                        _logger.LogError(
+                            "An error occurred while trying to create the table and index. {ErrorMessage}",
+                            ex.Message);
                         transaction.Rollback();
+
+                        return 1;
                     }
                 }
             }
         }
+
+        private void ValidateConnectionString(string _connectionString)
```

Comment:
Is it supposed to take the connection string as a parameter or use the class variable?